src/features/problem/ui/Tabs/Discussions/DiscussionContent.tsx

```
@@ -0,0 +1,103 @@
+  const { mutateAsync: editMutate, data: editResponse } = useEditDiscussionContent(
+    String(discussionContent.problemId),
+    discussionContent.discussionId
+  );
+  const { mutateAsync: deleteMutate, data: deleteResponse } = useDeleteDiscussionContent(
+    String(discussionContent.problemId),
+    discussionContent.discussionId
+  );
+
+  useEffect(() => {
+    if (editResponse && editResponse.data.success) {
+      setIsEdit(false);
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)